syz-manager/last_executing.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2024 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package main
+
+import (
+	"sort"
+	"time"
+)
+
+type LastExecuting struct {
```

Comment:
It would be nice to have a short comment re. what it is doing / what it is for.

Reply:
Done.